api/src/pcapi/core/offerers/commands.py

```diff
@@ -56,13 +56,16 @@ def check_active_offerers(dry_run: bool = False) -> None:
 
 
 @blueprint.cli.command("synchronize_venues_banners_with_google_places")
-@click.argument("begin", type=int, required=True)
-@click.argument("end", type=int, required=True)
+@click.argument("begin", type=int, required=False, default=0)
+@click.argument("end", type=int, required=False, default=None)
 @click.argument("limit", type=int, required=False, default=None)
-def synchronize_venues_banners_with_google_places(begin: int, end: int, limit: int | None = None) -> None:
+def synchronize_venues_banners_with_google_places(
+    begin: int = 0, end: int | None = None, limit: int | None = None
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏J'ai pas compris tout de suite que `begin` est un id de venue
proposition: `start_venue_id` et `end_venue_id`